src/core/respec.js

```diff
@@ -0,0 +1,24 @@
+// @ts-check
+/**
+ * This module adds a `respec` object to the `document` with the following
+ * readonly properties:
+ *  - version: returns version of ReSpec Script.
+ *  - ready: an alias to `document.respecIsReady`.
+ */
+
+export const name = "core/respec";
+
+class ReSpec {
```

Comment:
Creating a class here definitely looks weird, but it's due to next PR (`class ReSpec extends EventTarget`)